### ISSUE TITLE
Datepicker missing Month Left and Right Icons

### ISSUE THREAD

**Issue by jojoforce** [appiphony/appiphony-lightning-js#92]
I'm using the Datepicker component and it's not displaying the left and right arrow button to move between months. I am using the component on a visualforce page, any idea what could causing this issue?

My Screenshot
![image](https://user-images.githubusercontent.com/24530217/31913902-b37b209e-b816-11e7-8c64-6c37431e32f2.png)

Screenshot from http://aljs.appiphony.com/#!/datepickers
![image](https://user-images.githubusercontent.com/24530217/31913919-c818808c-b816-11e7-8acf-4c4610e91863.png)

**Comment by grivero** [comment]
I'm having the same issue, has anyone resolved this?

**Comment by grivero** [comment]
Ok, i was able to resolve the issue with a funky workaround.
This works for me because i'm importing the styles using the <apex:slds/> tag.
Essentially, we need to change the path to bring the images in the datepicker js file

from:
<use xlink:href="{{assetsLocation}}/assets/icons/utility-sprite/svg/symbols.svg#left"></use>
<use xlink:href="{{assetsLocation}}/assets/icons/utility-sprite/svg/symbols.svg#right"></use>

to:
/apexpages/slds/latest/assets/icons/utility-sprite/svg/symbols.svg#left
/apexpages/slds/latest/assets/icons/utility-sprite/svg/symbols.svg#right

because <use xlink:href="{{assetsLocation}}/assets/icons/utility-sprite/svg/symbols.svg#left"></use> uses https://devucsfhealth--dev--c.cs21.visual.force.com/assets/icons/utility-sprite/svg/symbols.svg 

but if you use the correct syntax suggested by SLDS documentation (<use xlink:href="{!URLFOR($Asset.SLDS, '/assets/icons/utility-sprite/svg/symbols.svg#left')}"></use>) the page renders /apexpages/slds/latest/assets/icons/utility-sprite/svg/symbols.svg#left.

Again, this works fine for me because i'm using the <apex:slds/> tag to import the styles.
I tried to use {!URLFOR($Asset.SLDS, inside the JS file but it didn't render correctly once the js was loaded (as it is reasonable)

Hope this can help you @jojoforce and all the people that came across this issue

**Comment by vecna0** [comment]
Did you read the docs?

<script>
    $.aljsInit({
        assetsLocation: assetsLocation,
        scoped: true,
        scopingClass: 'slds-scope'
    });
</script>

**Comment by jojoforce** [comment]
Yes I read the doc and I have that on my page. Your post did not help.

```

	<script> 
		$.aljsInit({ 
			assetsLocation: '{!URLFOR($Asset.SLDS)}', 
			scoped: true, 
			scopingClass: 'slds-scope' 
		});
	</script>
```
